.github/scripts/Release.py

```diff
@@ -261,23 +260,23 @@ def build_bibtex_entry(metadata: dict):
         for author in (metadata['Authors']['Core']['List'] +
                        metadata['Authors']['Developers']['List'])
     ] + [pybtex.database.Person("others")]
-    publication_date = datetime.date.fromisoformat(metadata['PublicationDate'])
     entry = pybtex.database.Entry(
         'software',
         persons=dict(author=authors),
         fields=dict(
-            title="{" + metadata['Name'] + "}",
-            # The 'version' field is not used by revtex4-2, so we use the 'note'
-            # field for now
-            # version=metadata['Version'],
-            note=r"\texttt{v" + metadata['Version'] + "}",
+            title=(r"\texttt{" + metadata['Name'] + " v" +
+                   metadata['Version'] + "}"),
+            # The 'version' field is not used by revtex4-2, so we also put the
+            # version in the title
+            version=metadata['Version'],
             publisher="Zenodo",
             doi=metadata['Doi'],
-            url="https://doi.org/" + metadata['Doi'],
-            howpublished=r"\url{" + metadata['Homepage'] + "}",
+            url=metadata['Homepage'],
+            howpublished=(r"\href{https://doi.org/" + metadata['Doi'] + "}{" +
+                          metadata['Doi'] + "}"),
```

Comment:
What styles use `howpublished`?  Do they ignore `doi`?

Reply:
All I can tell you is that it renders in PRD's `revtex4-2` like this:

<img width="385" alt="Bildschirmfoto 2022-01-03 um 23 30 41" src="https://user-images.githubusercontent.com/746230/147987439-1540410a-83fe-4d0a-83fa-5be01a20facf.png">

Reply:
Yeah, that's fine.  Looks good to me.

(The reason I've been pushing here is that I've had to go through .bib files fixing places where authors put doi info in multiple places and more than one was displayed.  I think the BibTeX designers screwed up trying to define a generic format but then leaving 90% of the meaning as implementation defined.)

Reply:
Yea I think the `@software` type is just an alias of `@misc` and there's no consistent standard how styles render it. So people will inevitably have to tweak it for the style their journal prefers, we can only give a recommendation.